### PR TITLE
LPS-168556 Users Contact add button should announce the group title

### PR DESCRIPTION
**Pull request by pat270**
LPS-168546 Users Contact use div instead of h# to prevent buttons from being read as "headling level".

https://issues.liferay.com/browse/LPS-168556
https://issues.liferay.com/browse/LPS-168546

### CODE REVIEW

**Comment by pat270** [comment]
ci:test:sf

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/common/addresses.jsp [hunk 1]
```diff
@@ -55,6 +54,7 @@ List<Address> addresses = AddressServiceUtil.getAddresses(className, classPK);
 							"classPK", classPK
 						).buildString()
 					%>'
+					label="add"
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/common/addresses.jsp:57]: Should this have `role="button"` like the others?

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one comment.

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/common/addresses.jsp (hunk 1 above)
Comment (pat270) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/common/addresses.jsp:57]: Yep it should, I updated it. Thanks @drewbrokke !

Comment (drewbrokke) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/common/addresses.jsp:57]: anytime!

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**